srcalc/src/main/java/gov/va/med/srcalc/domain/model/BooleanTerm.java

```diff
@@ -44,6 +46,14 @@ public void setVariable(BooleanVariable variable)
     }
     
     @Override
+    public double getSummand(final Value inputValue)
+    {
+        final boolean isTrue = Boolean.TRUE.equals(inputValue.getValue());
+        
+        return isTrue ? getCoefficient() : 0.0;
```

Comment:
You're right, I could do it in a single line, but the line would be long and having a separate variable facilitates debugging.